css/contact/style.css

```
@@ -39,5 +102,30 @@
     .contact-notes{
         font-size: 13px;
     }
+    .contact-container{
+        /* 中央コンテナ */
+        max-width: 348px;
+        margin: 0 auto;
+        padding: 0 20px;
+    }
+    
+    .form-row {
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 8px;
+    }
+    
+    .form-row label {
+        min-width: auto;
+    }
+    
+    .form-row input,
+    .form-row textarea {
+        font-size: 20px;
+    }
+    
+    .form-row textarea {
+        width: 331px;
+    }
```

カンプに合わせて、SPでは各フォームは幅いっぱいにするのがUXが上がるので、下記のように調整お願いします🙏
```suggestion
    .form-row dt,
    .form-row dd {
           width: 100%;
     } 
    .form-row input,
    .form-row textarea {
        font-size: 100%;
    }
    
    .form-row textarea {
        width: 100%;
    }
```

---

イメージもありがとう！めちゃわかりやすい🥹こちら調整しました！